openquake/smt/tests/comparison/data/compare_gmpe_inputs.toml

```diff
@@ -0,0 +1,113 @@
+### Input file for comparison of GMPEs using plotting functions in openquake.smt.comparison.compare_gmpes
+
+###For region parameter, we can use:
+#0 for global
+#1 for California
+#2 for Japan
+#3 for China
+#4 for Italy
+#5 for Turkey (locally = 3)
+#6 for Taiwan (locally = 0) 
+
+#Specify name of each run
+[run_names]
+name = ['Albania_1std','Albania_2std','Albania_3std']
+
+#Specify Vs30 for each run
+[vs30_values]
+vs30 = [800,800,600]
+
+#Specify Z1pt0 for each run
+[z1pt0_values]
+Z1 = [-999,-999,-999]
+
+#Specify z2pt5 for each run
+[z2pt5_values]
+Z25 = [-999,-999,-999]
+     
+#Specify region for each run 
+[region_list]
+region = [0,0,0]
+
+#Specify strike value for each run
+[strike_values]
+strike = [-999,-999,-999]
+
+#Specify dip value for each run
+[dip_values]
+dip = [-999,-999,-999]
+
+#Specify rake value for each run
+[rake_values]
+rake = [60,60,60]
+
+#Specify trellis mag values for each run for use in trellis plots (1 array per run)
+[trellis_mag_values]
+trellis_mag_list = [[5,6,7],[5,6,7],[5,6,7]]
+
+#Specify mag values for each run for use in Sammons, Euclidean distance matrix and clustering
+[mag_values]
+[mag_values.0]
+mmin = 5
+mmax = 7
+spacing = 0.1
+[mag_values.1]
+mmin = 5
+mmax = 7
+spacing = 0.1
+[mag_values.2]
+mmin = 5
+mmax = 7
+spacing = 0.1
+
+#Specify EQ depth value for each run
+[depth_values]
+depth = [30,30,30]
+
+#Specify maxR for each run
+[maxR_values]
+maxR = [300,300,300]
+
+#Specify number of GMPE sigma to sample from per run
+[no_std_per_run]
+Nstd = [1,2,3]
+
+#Specify imt_list for each run
+[imts_list]
+[imt_list.0] 
+imt_list = ['PGA','SA(0.1)','SA(0.5)','SA(1.0)']
+[imt_list.1]
+imt_list = ['PGA','SA(0.1)','SA(0.5)','SA(1.0)']
+[imt_list.2]
+imt_list = ['PGA','SA(0.1)','SA(0.5)','SA(1.0)','SA(2.0)']
+
+#Specify gmpe label for each run
+[gmpes_label]
+[gmpes_label.0]
+gmpes_label=['CY14','CB14','BSSA14','KO2020']
+[gmpes_label.1]
+gmpes_label=['CY14','CB14','BSSA14','KO2020']
+[gmpes_label.2]
+gmpes_label=['CY14','CB14','BSSA14','KO2020']
+
+
+#Specify gmpes for each run
+[models] 
+
+[models.0]
```

Comment:
Why so many numbers? Maybe you could add as a comment a short description explaining the format

Reply:
> Why so many numbers? Maybe you could add as a comment a short description explaining the format

 Chris, on the comparison module, I think that the input toml is too complicated. My initial csv had different lines for different analyses (for instance different tectonic context in the same hazard model). However I think that when transformed in  toml (what you call “run” it becomes a bit messy. i’d suggest we have only one “run” and if a user wants to do more tests, they will create more tomls, does it make sense?